receiver/otlpreceiver/otlp.go

```diff
@@ -16,11 +16,12 @@ package otlpreceiver
 
 import (
 	"context"
+	"io/ioutil"
 	"net"
 	"net/http"
 	"sync"
 
-	gatewayruntime "github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/gorilla/mux"
```

Comment:
grpc-gateway changes should maybe be in separate PR?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/3422